auth/auth.go

```diff
@@ -118,6 +118,6 @@ func (m *userAgentMiddleware) HandleBuild(ctx context.Context, in middleware.Bui
 	if !ok {
 		return next.HandleBuild(ctx, in)
 	}
-	req.Header.Set("User-Agent", m.providerName + "/" + ProviderVersion + " " + req.Header.Get("User-Agent"))
+	req.Header.Set("User-Agent", m.providerName+"/"+ProviderVersion+" "+req.Header.Get("User-Agent"))
```

Comment:
nit: You could use [`req.Header.Add()`](https://pkg.go.dev/net/http#Header.Add) instead to append to the user agent list instead of injecting the old one